prototypes/as2/gradlew.bat

```diff
@@ -1,89 +1,89 @@
-@rem
```

Comment:
No changes to this file and the one above.  These showed up in the PR after a merge from master.